contracts/SuperpowerNFTBase.sol

```diff
@@ -266,7 +273,13 @@ abstract contract SuperpowerNFTBase is
     emit ForcefullyUnlocked(tokenId);
   }
 
-  // manage approval
+  // TO obtain the lockability, the standard approval and transfer
```

Comment:
minor typo in comments "TO" should be "To"

Reply:
Fixed